Formula/tutum.rb

```diff
@@ -1,8 +1,8 @@
 class Tutum < Formula
   desc "Docker platform for dev and ops"
   homepage "https://www.tutum.co/"
-  url "https://github.com/tutumcloud/cli/archive/v0.21.0.tar.gz"
-  sha256 "7657b337292cd480cd206776647bf6eee8c4e7b7ec370f42d2c2685818752e70"
+  url "https://github.com/tutumcloud/cli/archive/v0.21.1.tar.gz"
```

Comment:
I think the pip upload is supposed to be optimised to some degree, but I could be misremembering that from something else entirely. CC @tdsmith